### PR TITLE
adding .secrets.baseline

### PR DESCRIPTION
**Pull request by domanchi**

### CODE REVIEW

File: .secrets.baseline [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    },
+    {
+      "hex_limit": 3,
+      "name": "HexHighEntropyString"
+    },
+    {
+      "name": "PrivateKeyDetector"
+    }
+  ],
+  "results": {},
```
Comment (chriskuehl) [inline .secrets.baseline]: If the baseline is empty, can we avoid committing this file?

Comment (domanchi) [reply .secrets.baseline]: It doesn't find anything, but it also contains configuration details of the scan.

Comment (asottile) [reply .secrets.baseline]: isn't an unscanned repo with no secrets the same as a scanned repo with no secrets?

Comment (domanchi) [reply .secrets.baseline]: Yeah, but we wouldn't know whether an unscanned repo has no secrets, until we scan it.

So,

1. If there's no baseline, it's unscanned.
2. If it's unscanned, we don't know its status.

Comment (asottile) [reply .secrets.baseline]: wouldn't you just scan it in that case?

**Pull request closed without merging**